source/PlayerInfo.cpp

```diff
@@ -2110,6 +2130,63 @@ void PlayerInfo::SetTravelDestination(const Planet *planet)
 
 
 
+void PlayerInfo::QueueRelocation(const Planet *destination, bool flagshipOnly)
+{
+	relocationPlanet = destination;
+	flagshipOnly = flagshipOnly;
```

Comment:
```suggestion
	relocateFlagshipOnly = flagshipOnly;
```
This is why flagship only wasnt working